### PR TITLE
Add formula for OpenTSDB

### PR DESCRIPTION
**Pull request by florinbroasca**

### CODE REVIEW

File: Library/Formula/opentsdb.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
+class Opentsdb < Formula
+  homepage "http://opentsdb.net"
+  url "https://github.com/OpenTSDB/opentsdb/releases/download/v2.1.0/opentsdb-2.1.0.tar.gz"
+  sha256 "b76e36a2d744170dde0ecd7ff3b937cbe68dd9fab9ce885aab5847de3e975157"
+
+  def install
+    ENV.deparallelize
```
Comment (mikemcquaid) [inline Library/Formula/opentsdb.rb]: Can you please submit this issue to the upstream developers of this project and add a link to the upstream issue submission in a comment in the formula file. Thanks!

File: Library/Formula/opentsdb.rb [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    Dir.chdir("build") do
+      system "make", "install"
+    end
+  end
+
+  test do
+    system "true"
+  end
+
+  patch :p0, :DATA
```
Comment (mikemcquaid) [inline Library/Formula/opentsdb.rb]: Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!

File: Library/Formula/opentsdb.rb [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    ENV.deparallelize
+    system "./build.sh --prefix=#{prefix}"
+    system "mkdir -p #{prefix}/share/opentsdb/static/gwt/standard/images/ie6"
+    Dir.chdir("build") do
+      system "make", "install"
+    end
+  end
+
+  test do
+    system "true"
```
Comment (mikemcquaid) [inline Library/Formula/opentsdb.rb]: Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

**Comment by mikemcquaid** [comment]
Please fix the `brew audit --strict opentsdb` warnings, thanks: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/26756/version=yosemite/testReport/junit/brew-test-bot/yosemite/audit_opentsdb___strict/

**Comment by DomT4** [comment]
Mike's comments need addressing here, Thanks!

**Comment by DomT4** [comment]
There's been no activity here since the PR was submitted, even to comments from Mike on the day. Closing consequently.

Please feel free to submit an updated PR if you'd like to, Thanks for the submission!

**Pull request closed without merging**